packages/react-components/react-utilities/src/hooks/useControllableState.test.ts

```diff
@@ -1,9 +1,9 @@
 import { renderHook, act } from '@testing-library/react-hooks';
-import { useControllableState } from './useControllableState';
+import { UseControllableStateOptions, useControllableState } from './useControllableState';
 
 type RenderProps<T> = {
   state: boolean | undefined;
-  defaultState?: boolean;
+  defaultState: boolean | undefined;
```

Comment:
Just curious, why change this to `| undefined`?

Reply:
To be sure it is defined! the value of the variable can be `undefined`, but the variable must be defined!

`?` question mark indicates you can avoid defining `defaultState` value, which in most cases is erroneous, as there should be a `props.defaultValue` property to allow the user the possibility to provide an initial state to a uncontrolled value.